### ISSUE TITLE
Logic-view separation

### ISSUE THREAD

**Issue by xvw** [Wano-k/3D-Game-Engine-csharp#5]
You should not use your view for logic. 
For example, you can create a class Project with two static methods : 
-  Load (return a new filled project with his data) 
-  Create (create folders and return the filled project)
  And use it into your views. Using a class for project representation give you a local scope for general data.

**Comment by Wano-k** [comment]
Thanks! I wanted to change it. I tried to do that kind of idea with the new project dialog: 
- class Project which is the main model with all the values (here the project name, and location)
- class DialogNewProjectControl implementing INotifyPropertyChanged interface. Here are all the functions I can use when I handle an event (the methods you suggested for example). I use NotifyPropertyChanged function into auto property set which is updating the specific view when I'm updating a model property. I deleted the "using System.Windows.Forms" in order to be sure that I'm not using any view items.
- class DialogNewProject is the view handling envents and using the controller for any logic.

Is that a great approach now? (I pushed my solution)
